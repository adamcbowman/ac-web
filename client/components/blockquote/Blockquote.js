import React from 'react'
import PropTypes from 'prop-types'
import CSSModules from 'react-css-modules'
import styles from './Blockquote.css'

Blockquote.propTypes = {
    children: PropTypes.node.isRequired,
}

function Blockquote({ children }) {
    return (
        <blockquote styleName="Blockquote">
            {children}
        </blockquote>
    )
}

export default CSSModules(Blockquote, styles)
