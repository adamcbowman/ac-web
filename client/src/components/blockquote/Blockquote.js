import React from 'react'
import CSSModules from 'react-css-modules'
import styles from './Blockquote.css'

function Blockquote({children}) {
    return (
        <blockquote styleName='Blockquote'>
            {children}
        </blockquote>
    )
}

export default CSSModules(Blockquote, styles)
