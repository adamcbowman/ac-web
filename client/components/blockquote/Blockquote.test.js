import React from 'react'
import Renderer from 'react-test-renderer'
import Blockquote from './Blockquote'

test('blockquote component', () => {
    const blockquote = Renderer.create(
        <Blockquote>Blockquote content</Blockquote>
    )

    expect(blockquote).toMatchSnapshot()
})
