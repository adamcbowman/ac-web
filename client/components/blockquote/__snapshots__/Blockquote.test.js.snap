// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blockquote component 1`] = `
<blockquote
  className="Blockquote"
>
  Blockquote content
</blockquote>
`;
